manifests/0000_50_cluster-monitoring-operator_04-config.yaml

```diff
@@ -467,9 +467,47 @@ data:
     #
     # owners: (@openshift/networking)
     #
-    # ovnkube_master_egress_routing_via_host" informs if the OVN-K cluster's gateway mode is
+    # cluster:ovnkube_controller_egress_routing_via_host:max informs if the OVN-K cluster's gateway mode is
     # `routingViaOVN` (0), `routingViaHost` (1) or invalid (2).
-    - '{__name__="cluster:ovnkube_master_egress_routing_via_host:max"}'
+    - '{__name__="cluster:cluster:ovnkube_controller_egress_routing_via_host:max"}'
```

Comment:
```suggestion
    - '{__name__="cluster:ovnkube_controller_egress_routing_via_host:max"}'
```

Reply:
yep done! very nice catch thank you, while fixing conflicts on 4.16 it seems this got into a copy-paste snafu